ansible/roles/load_test/templates/pgbench-test.j2

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# {{ ansible_managed }}
+#
+
+scales=( 1 10 100 1000 3000 5000 10000 )
+start=$(date)
+for i in `seq {{ iterations }}`
+do
+    for j in `seq {{ reruns }}`
+    do
+        out=""
+        conn_options=" -U postgres -h {{ groups['load_test_host'][0] }} -p 5432 "
+
+        # Initialize database
+        scale=$((scales[$i - 1]))
+        pgbench{{ postgresql_version }} $conn_options -i -s $scale {{ db_name }}
+
+        # Run actual test
+        {% if item == "read" %}
+        out+=$(pgbench{{ postgresql_version }} $conn_options -c {{ connections }} -j {{ threads }} -T {{ time_in_sec }} -S {{ db_name }})
+        {% elif item == "writeread" %}
+        out+=$(pgbench{{ postgresql_version }} $conn_options -c {{ connections }} -j {{ threads }} -T {{ time_in_sec }} {{ db_name }})
```

Comment:
Great, thanks. Based on the graph you sent out previously I think the wiki formula will put us somewhere in the right zone. I did the calc yesterday and came up with a scale of ~4500 for 'mostly cache'